dev.sh

```
@@ -1,69 +1,200 @@
-else
-    echo "✅ No MCP server processes found on port $MCP_PORT"
-fi
+# Function to stop servers
+stop_servers() {
+    echo "🛑 Stopping servers..."
+    
+    # Find and kill API server processes
+    API_PIDS=$(lsof -ti:$API_PORT 2>/dev/null)
+    if [ -n "$API_PIDS" ]; then
+        echo "🛑 Killing API server processes on port $API_PORT: $API_PIDS"
+        kill -9 $API_PIDS
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)